clippy_dev/src/lint.rs

```diff
@@ -36,20 +36,12 @@ pub fn run<'a>(path: &str, args: impl Iterator<Item = &'a String>) {
     } else {
         exit_if_err(Command::new("cargo").arg("build").status());
 
-        // Run in a tempdir as changes to clippy do not retrigger linting
-        let target = tempfile::Builder::new()
```

Comment:
This is great! Do we still use the `tempfile` dep anywhere or can this be removed?

Reply:
Seems not, removed it